multicluster/docs/getting-started.md

```diff
@@ -0,0 +1,257 @@
+3. Copy the image file `antrea-mcs.tar` to the nodes of your local cluster
+4. Run `docker load < antrea-mcs.tar` in each node of your local cluster.
+
+## Install MCS
+
+After you prepare MCS docker image, you can run below command to install MCS component:
+
+```shell
+kubectl apply -f multicluster/config/multi-cluster.yaml
+```
+
+## Setup ClusterSet
+
+In an Antrea multi-cluster cluster set, there will be at least one leader cluster and one member cluster. At first, all clusters in the cluster set need to use `ClusterClaim` to claim itself as a member of a cluster set. A leader cluster will define `ClusterSet` which includes leader and member clusters. below is a sample to create a cluster set with a cluster set id `test-clusterset` which has two member clusters with cluster id `test-cluster-east` and `test-cluster-west`, one leader cluster with id `test-cluster-leader`, if you'd like to set up an MCS cluster set with two clusters, you can use one of member cluster `test-cluster-east` or `test-cluster-west` to replace `test-cluster-leader`.
+
+* Create below `ClusterClaim` in the member cluster `test-cluster-east`.
+
+```yaml
+apiVersion: multicluster.crd.antrea.io/v1alpha1
+kind: ClusterClaim
+metadata:
+  name: east-membercluster-id
+  namespace: antrea-mcs-ns
+name: id.k8s.io
+value: test-cluster-east
+---
+apiVersion: multicluster.crd.antrea.io/v1alpha1
+kind: ClusterClaim
+metadata:
+  name: clusterset-id
+  namespace: antrea-mcs-ns
+name: clusterSet.k8s.io
+value: test-clusterset
+```
+
+* Create below `ClusterClaim` in the member cluster `test-cluster-west`.
+
+```yaml
+apiVersion: multicluster.crd.antrea.io/v1alpha1
+kind: ClusterClaim
+metadata:
+  name: west-membercluster-id
+  namespace: antrea-mcs-ns
+name: id.k8s.io
+value: test-cluster-west
+---
+apiVersion: multicluster.crd.antrea.io/v1alpha1
+kind: ClusterClaim
+metadata:
+  name: clusterset-id
+  namespace: antrea-mcs-ns
+name: clusterSet.k8s.io
+value: test-clusterset
+```
+
+* Create below `ClusterClaim` in the leader cluster `test-cluster-leader`.
+ 
+```yaml
+apiVersion: multicluster.crd.antrea.io/v1alpha1
+kind: ClusterClaim
+metadata:
+  name: leadercluster-id
+  namespace: antrea-mcs-ns
+name: id.k8s.io
+value: test-cluster-leader
+---
+apiVersion: multicluster.crd.antrea.io/v1alpha1
+kind: ClusterClaim
+metadata:
+  name: clusterset-id
+  namespace: antrea-mcs-ns
+name: clusterSet.k8s.io
+value: test-clusterset
+```
+
+* Create below `ClusterSet` in the leader cluster `test-cluster-leader`.
+
+```yaml
+apiVersion: multicluster.crd.antrea.io/v1alpha1
+kind: ClusterSet
+metadata:
+    name: test-clusterset
+    namespace: kube-system
+spec:
+    leaders:
+      - clusterID: test-cluster-leader
+        secret: "leader-access-token"
+        server: "https://172.18.0.2:6443"
+    members:
+      - clusterID: test-cluster-east
+      - clusterID: test-cluster-west
+    namespace: antrea-mcs-ns
+```
+
```

Comment:
Need to add help about creating sa and associated role-binding, i can help with that after this is merged

Reply:
hi @aravindakidambi  I have addressed your comments, please help to double check, if it's ok, I think you can merge and update sa and role-binding parts. thanks.